### ISSUE TITLE
Powershell update button uses old brew install syntax that is deprecated

### ISSUE THREAD

**Issue by philippkahr** [PowerShell/vscode-powershell#3131]
### System Details

System Details Output
```
PS /Users/philipp> & {"### VSCode version: $(code -v)"; "`n### VSCode extensions:`n$(code --list-extensions --show-versions | Out-String)"; "`n### PSES version: $($pseditor.EditorServicesVersion)"; "`n### PowerShell version:`n$($PSVersionTable | Out-String)"}

### VSCode version: 1.52.1 ea3859d4ba2f3e577a159bc91e3074c5d85c0523 x64

### VSCode extensions:
dbaeumer.vscode-eslint@2.1.14
eamodio.gitlens@11.1.3
esbenp.prettier-vscode@5.8.0
James-Yu.latex-workshop@8.15.0
mechatroner.rainbow-csv@1.8.0
ms-azuretools.vscode-docker@1.9.0
ms-python.python@2020.12.424452561
ms-toolsai.jupyter@2020.12.414227025
ms-vscode-remote.remote-containers@0.154.2
ms-vscode.powershell@2020.6.0
redhat.vscode-yaml@0.14.0
VisualStudioExptTeam.vscodeintellicode@1.2.10
vscode-icons-team.vscode-icons@11.1.0
yzhang.markdown-all-in-one@3.4.0


### PSES version: 2.2.0.0

### PowerShell version:

Name                           Value
----                           -----
PSVersion                      7.1.0
PSEdition                      Core
GitCommitId                    7.1.0
OS                             Darwin 20.2.0 Darwin Kernel Version 20.2.0: Wed Dec  2 20:39:59 PST 2020; root:xnu-7195.60.75~1/RELEASE_X86_64 (MacOS 11)
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Brew version

```
PS /Users/philipp> brew config

HOMEBREW_VERSION: 2.7.4
ORIGIN: https://github.com/Homebrew/brew
HEAD: 663b972f449b679512a2c0e3303569a2e9bfba7c
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 800e4b5fec9e6047fd0a1100500398cf85834247
Core tap last commit: 44 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.FaFo3wfVHI/org.macosforge.xquartz:0
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
```

### Issue Description

I am experiencing a problem with the auto update toast notification. It uses the old `brew upgrade cask powershell` which is not supported anymore. Instead, it is `brew upgrade --cask powershell`.

#### Expected Behaviour

Use `brew upgrade --cask powershell`

#### Actual Behaviour

```
PS /Users/philipp> brew cask upgrade powershell
Error: Calling brew cask upgrade is disabled! Use brew upgrade [--cask] instead.
```

![Screenshot 2021-01-15 at 13 28 22](https://user-images.githubusercontent.com/12175559/104727738-25dca600-5736-11eb-9491-60c8cdabcc25.png)
![Screenshot 2021-01-15 at 13 28 28](https://user-images.githubusercontent.com/12175559/104727741-270dd300-5736-11eb-9973-766188cc9e21.png)

**Comment by SydneyhSmith** [comment]
@philippkahr thanks for opening the issue and the PR! We may want to update this install system in the future to not have to stay up to date with brew

**Comment by andschwa** [comment]
The fix for this was merged in #3132, not sure why the issue didn't close.
